portfolio/src/main/webapp/script.js

```diff
@@ -36,15 +36,29 @@ function randQuote() {
   }
 }
 
-// Get fetch 'Welcome' from DataServlet.java
-function getWelcome() {
-  fetch('/welcome').then(response => response.text()).then((welcome_str) => {
-    document.getElementById("fetch_welcome").innerHTML = welcome_str;
+// Fetch comments from DataServlet.java
+function getComments() {
+
+  const str_comments_intro = "<h2>Comments</h2><p>";
+
+  fetch('/comments').then(response => response.json()).then((comments) => {
+
+    const str_comments = str_comments_intro.concat(
```

Comment:
Can you update the logic here to not depend `comments` being a list of size 3 (i.e., can you loop through `comments` instead)? 

That way, this code wouldn't have to change if you were to add more hard-coded comments.

Reply:
Ok